src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -61,11 +61,14 @@ sealed trait Task {
 
   def runSpecId: PathId = taskId.runSpecId
 
+  def status: Task.Status
+
   def launchedMesosId: Option[MesosProtos.TaskID] = launched.map { _ =>
     // it doesn't make sense for an unlaunched task
     taskId.mesosTaskId
   }
 
+  // TODO ju remove
```

Comment:
Please remove the TODO for now and create a follow up ticket containing removed TODO's